ofl/alegreya/METADATA.json

```diff
@@ -8,63 +8,61 @@
-      "style": "normal",
+      "postScriptName": "Alegreya-BoldItalic",
+      "fullName": "Alegreya Bold Italic",
+      "style": "italic",
       "weight": 700,
-      "filename": "Alegreya-Bold.ttf",
-      "postScriptName": "Alegreya-Bold",
-      "fullName": "Alegreya Bold",
-      "copyright": "Copyright (c) 2011, Juan Pablo del Peral (juan@huertatipografica.com.ar), with Reserved Font Names \"Alegreya\" \"Alegreya SC\""
+      "filename": "Alegreya-BoldItalic.ttf",
+      "copyright": "Copyright (c) 2011, Juan Pablo del Peral (juan@huertatipografica.com.ar), with Reserved Font Names \"Alegreya\" \"Alegreya SC\". Copyright (c) 2013 by Adobe Systems Incorporated. All Rights Reserved."
     },
     {
       "name": "Alegreya",
-      "style": "italic",
-      "weight": 700,
-      "filename": "Alegreya-BoldItalic.ttf",
-      "postScriptName": "Alegreya-BoldItalic",
-      "fullName": "Alegreya Bold Italic",
-      "copyright": "Copyright (c) 2011, Juan Pablo del Peral (juan@huertatipografica.com.ar), with Reserved Font Names \"Alegreya\" \"Alegreya SC\""
+      "postScriptName": "Alegreya-Regular",
+      "fullName": "Alegreya",
+      "style": "normal",
+      "weight": 400,
+      "filename": "Alegreya-Regular.ttf",
+      "copyright": "Copyright (c) 2011, Juan Pablo del Peral (juan@huertatipografica.com.ar), with Reserved Font Names \"Alegreya\" \"Alegreya SC\". Copyright (c) 2013 by Adobe Systems Incorporated. All Rights Reserved."
     },
     {
       "name": "Alegreya",
+      "postScriptName": "Alegreya-Black",
+      "fullName": "Alegreya Black",
       "style": "normal",
       "weight": 900,
       "filename": "Alegreya-Black.ttf",
-      "postScriptName": "Alegreya-Black",
-      "fullName": "Alegreya Black",
-      "copyright": "Copyright (c) 2011, Juan Pablo del Peral (juan@huertatipografica.com.ar), with Reserved Font Names \"Alegreya\" \"Alegreya SC\""
+      "copyright": "Copyright (c) 2011, Juan Pablo del Peral (juan@huertatipografica.com.ar), with Reserved Font Names \"Alegreya\" \"Alegreya SC\". Copyright (c) 2013 by Adobe Systems Incorporated. All Rights Reserved."
     },
     {
       "name": "Alegreya",
-      "style": "italic",
-      "weight": 900,
-      "filename": "Alegreya-BlackItalic.ttf",
-      "postScriptName": "Alegreya-BlackItalic",
-      "fullName": "Alegreya Black Italic",
-      "copyright": "Copyright (c) 2011, Juan Pablo del Peral (juan@huertatipografica.com.ar), with Reserved Font Names \"Alegreya\" \"Alegreya SC\""
+      "postScriptName": "Alegreya-Bold",
+      "fullName": "Alegreya Bold",
+      "style": "normal",
+      "weight": 700,
+      "filename": "Alegreya-Bold.ttf",
+      "copyright": "Copyright (c) 2011, Juan Pablo del Peral (juan@huertatipografica.com.ar), with Reserved Font Names \"Alegreya\" \"Alegreya SC\". Copyright (c) 2013 by Adobe Systems Incorporated. All Rights Reserved."
     }
   ],
   "subsets": [
-    "latin",
-    "latin-ext",
-    "menu"
+    "latin"
```

Comment:
Also it seems weird that the size field at beginning of the file did not change.